app/javascript/dashboard/modules/widget-preview/components/WidgetBody.vue

```diff
@@ -57,23 +121,26 @@ $tailwind-black-700: #3c4858;
       max-width: 100%;
       .chat-bubble {
         box-shadow: var(--shadow-medium);
-        background: var(--color-woot);
-        border-radius: var(--border-radius-large);
+        border-radius: 2rem;
         color: var(--white);
         display: inline-block;
         font-size: var(--font-size-nano);
         line-height: 1.5;
-        padding: var(--space-small) var(--space-one);
+        padding: 1.3rem 1.75rem;
         text-align: left;
-        word-break: break-word;
-        max-width: 100%;
+
+        p {
+          margin: var(--space-zero);
+        }
+
         &.user {
           border-bottom-right-radius: var(--border-radius-small);
+          background: var(--color-woot);
         }
         &.agent {
           background: var(--white);
           border-bottom-left-radius: var(--border-radius-small);
-          color: $tailwind-black-700;
+          color: black;
```

Comment:
Can we use variables here?
```suggestion
          color: var(--b-900);
```

Reply:
Done